packages/Button/src/Button.js

```diff
@@ -131,7 +131,7 @@ const Button = React.forwardRef((props, ref) => {
     ...moreProps,
   };
   if (isSemantic) {
-    buttonProps.disabled = isDisabled;
+    buttonProps.disabled = isDisabled || isPending;
```

Comment:
Good, but this only affects semantic buttons.  For `isSemantic={false}` buttons, you'll need to adjust line 126 as well.

Reply:
Ok made a few more adjustments in here as well to deal with isPending showing button in disabled state